test/e2e/v3/generate_test.go

```diff
@@ -229,3 +229,164 @@ Count int `+"`"+`json:"count,omitempty"`+"`"+`
 		_ = pluginutil.RunCmd("Update dependencies", "go", "mod", "tidy")
 	}
 }
+
+// GenerateV3 implements a go/v3(-alpha) plugin project defined by a TestContext.
+func GenerateV3WithKustomizeV2(kbc *utils.TestContext, crdAndWebhookVersion string) {
```

Comment:
Here we have the test to check the new plugin scaffold working on the cluster with go/v3 one.